src/main/java/uk/gov/justice/digital/hmpps/whereabouts/controllers/AbsentReasonsController.kt

```diff
@@ -16,8 +26,22 @@ class AbsentReasonsController {
   @GetMapping
   fun reasons(): AbsentReasonsDto =
     AbsentReasonsDto(
-      AbsentReason.paidReasons,
-      AbsentReason.unpaidReasons,
-      AbsentReason.iepTriggers
+      paidReasons,
+      unpaidReasons,
+      iepTriggers,
+    )
+
+  @GetMapping(value = ["/v2"])
```

Comment:
for now is new endpoint - will get DPS to switch over to using the endpoint, then remove old and rename v2 -> v